Backpack-SwiftUI/SearchControlInput/Classes/SearchControlInput.swift

```diff
@@ -0,0 +1,63 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2022 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+/// A control that displays an editable text interface for search inputs.
+public struct BPKSearchControlInput: View {
+    
+    private let label: String
+    private let icon: BPKIcon
+    
+    /// Creates a `BPKSearchControlInput`.
+    ///
+    /// - Parameters:
+    ///   - label: The t text to display when the text field is empty.
```

Comment:
there's a typo here with an extra `t`. But also its a bit misleading, It's not the text to be shown when the text field is empty, this text will always be shown (looking at the implementation at least). Keep in mind that the component does not know how it will be used so if it'll be connected to a textfield somewhere in the client, that should be transparent for this component

Reply:
what I meant is, this component does not know/care if the label is used as a label or a placeholder :)